src/Common/Infrastructure/Controllers/RegistryApiController.cs

```diff
@@ -36,7 +36,13 @@ public abstract class RegistryApiController<T> : ApiController<T>
             ILogger<T> logger) : base(redis, logger)
         {
             _restClient = restClient;
-            CacheToggle = cacheToggle;
+            _cacheToggle = cacheToggle;
+        }
+
+        protected bool ShouldGetFromCache()
+        {
+            return _cacheToggle.FeatureEnabled &&
+                   HttpContext.Request.Headers.CacheControl.ToString().ToLower() != "no-cache";
```

Comment:
`HttpContext.Request.Headers.CacheControl.Any(x => x.Contains("no-cache"))` something like this?

Reply:
We had to use GetCommaSeparatedValues. This handles following scenario's:
1. Cache-Control: value1, value2 , value3 (redundant spaces)
2. Cache-Control: value1
    Cache-Control: value2 (multiple entries for the same header)
ref. https://github.com/json-api-dotnet/JsonApiDotNetCore/pull/973